bin/oneoff/census/ap-data/check-school-codes.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../../dashboard/config/environment'
```

Comment:
Please add a short summary of what this script does, and/or add it to the `README.md`

Reply:
Oops, missed that one. I've added a note to the README.